tests/integration_tests/clouds.py

```diff
@@ -300,8 +290,7 @@ def _perform_launch(self, launch_kwargs, **kwargs):
         except KeyError:
             profile_list = self._get_or_set_profile_list(release)
 
-        prefix = datetime.datetime.utcnow().strftime("cloudinit-%m%d-%H%M%S")
-        default_name = prefix + "".join(
+        default_name = self.cloud_instance.tag + "".join(
```

Comment:
Maybe in pycloudlib we should make LXD use the `tag` to set the instances default name, like in the other clouds. (Not to be fixed here.)

Reply:
I think that makes sense in pycloudlib so all clouds behave the same-ish,